dev/scalar-galaxy/index.md

```
@@ -0,0 +1,16 @@
+---
+aside: true
+outline: [1, 2]
+title: vitepress-theme-openapi
+---
+
+<script setup lang="ts">
```

Markdownlint

Element: script
Inline HTML

(MD033, no-inline-html)